src/tools/ceph_objectstore_tool.cc

```diff
@@ -45,7 +45,6 @@
 #include "include/util.h"
 
 namespace po = boost::program_options;
-using namespace std;
```

Comment:
@shinobu-x i think there is an other `using namespace std` lurking somewhere else. because we can use `vector` w/o the `std::` in this very source file. and i think it's fine to `using namespace std` in a `.cc` file.